SA/sa.typ

```diff
@@ -3,13 +3,25 @@
 #set text(..sa_text_style)
 #set heading(..sa_header_style)
 
-#sa_title_page(
-  "VisualFP",
-  "Designing a Visual, Block-Based Environment to Create & Execute Haskell Code");
+#let metadata = (
+  title: [VisualFP],
+  description: [Designing a Visual, Block-Based Environment to Create & Execute Haskell Code],
+  organization: [Department of Computer Science \ OST - University of Applied Sciences \ Campus Rapperswil-Jona],
+  thesis: [Student Thesis],

```

Comment:
```suggestion
  thesis: [Term Paper],

```
The DeepL translation of "Studienarbeit" would be student research project, but I like the less common translation "term paper" better

Reply:
Sounds good 👍